blocks/hero/hero.js

```diff
@@ -0,0 +1,61 @@
+// Description: Hero block
+/**
+ * Inserts a node after a reference node
+ * @param {Element} referenceNode
+ * @param {Element} newNode
+ */
+function insertAfter(referenceNode, newNode) {
```

Comment:
The function can be possibly replaced by one of the standard methods:
- https://developer.mozilla.org/en-US/docs/Web/API/Element/after
- https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement

Reply:
done. I removed the code since I found another approach.